M2/Macaulay2/packages/AbstractSimplicialComplexes.m2

```diff
@@ -0,0 +1,832 @@
+-- -*- coding: utf-8 -*-
+--------------------------------------------------------------------------------
+-- Copyright 2024  Nathan Grieve
+--
+-- This program is free software: you can redistribute it and/or modify it under
+-- the terms of the GNU General Public License as published by the Free Software
+-- Foundation, either version 3 of the License, or (at your option) any later
+-- version.
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+-- details.
+--
+-- You should have received a copy of the GNU General Public License along with
+-- this program.  If not, see <http://www.gnu.org/licenses/>.
+--------------------------------------------------------------------------------
+
+
+newPackage(
+    "AbstractSimplicialComplexes",
+    Version => "0.1",
+    Date => "24 September 2024",
+    Headline => "AbstractSimplicialComplexes",
```

Comment:
The `Headline` should be probably be something more like "abstract simplicial complexes"